gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js

```diff
@@ -3451,7 +3452,8 @@ GbGradeTable.focusColumnForAssignmentId = function(assignmentId, showPopupForNew
                   });
               }
             })
-            (new bootstrap.Popover($selectedField[0])).show();
+            var p = new bootstrap.Popover($selectedField[0]);
+            p.show();
```

Comment:
```suggestion
            new bootstrap.Popover($selectedField[0]).show();
```

Reply:
That works. I just tried it on your branch.

Reply:
Agreed. I'll force push an edit momentarily.